src/main/kotlin/racingcar/Main.kt

```diff
@@ -1,14 +1,11 @@
 package racingcar
 
 fun main() {
-    val inputView = InputView()
-    inputView.excute()
-
     val race = Race()
 
-    race.start(inputView.carCount, inputView.roundCount)
+    val result = race.result(InputView.getCarNames(), InputView.getRoundCount())
```

Comment:
```suggestion
    val result = Race().result(
        carNames = InputView.getCarNames(),
        roundCount = InputView.getRoundCount()
    )
```
race 변수를 재활용하지 않으므로, 객체를 바로 생성해서 함수를 호출해도 좋을 것 같아요.
추가로, 파라미터가 어떤 것을 의미하는지 알기 어렵다면 argument name을 활용하면 가독성이 좋아집니다.